### PR TITLE
Code block

### PR DESCRIPTION
**Pull request by skliarzfort**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: js/customscripts.js [hunk 1]
```diff
@@ -22,16 +22,39 @@ $( document ).ready(function() {
...
+    initCopyText();
+});
+
+function initCopyText() {
+    jQuery('code').on('click', function(e){
+    let copyText = this,
+        textArea = document.createElement("textarea");
+
+        textArea.value = copyText.textContent.trim();
+        textArea.style.position = "fixed";
```
Comment (tolerants) [inline js/customscripts.js:38]: Please move styles to css class and add/remove it.

File: js/customscripts.js [hunk 2]
```diff
@@ -22,16 +22,39 @@ $( document ).ready(function() {
...
+    initSidebarAccordion();
+
+    initFeedbackForm();
+
+    initCopyText();
+});
+
+function initCopyText() {
+    jQuery('code').on('click', function(e){
+    let copyText = this,
```
Comment (tolerants) [inline js/customscripts.js:34]:
```suggestion
        let copyText = this,
```

File: js/toc.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // https://github.com/ghiculescu/jekyll-table-of-contents
 (function($){
+  return;
```
Comment (tolerants) [inline js/toc.js:3]: Do you just forgot to remove it?

**Review by tolerants — CHANGES_REQUESTED** [review]

File: _layouts/page.html [hunk 4]
```diff
@@ -53,6 +53,8 @@ <h1 class="post-header__title">{{ page.title }}</h1>
             {% endif %}
         </div>
 
+        <div class="post-meta">Last updated: <time datetime="">20.05.2021</time></div>
```
[the remark this answers is not in the document]
Comment (skliarzfort) [reply _layouts/page.html:56]: I added static code as we discussed with Dmitriy

File: js/customscripts.js (hunk 2 above)
Comment (skliarzfort) [reply js/customscripts.js:34]: let's skip it. script was totally changed in next branch

**Review by tolerants — APPROVED** [review]

**Pull request merged**